### ISSUE TITLE
Quick question

### ISSUE THREAD

**Issue by lcnogueira** [stefan-prokop-cz/verify-apple-id-token#3]
hi, @stefan-prokop-cz 👋 . First of all, thanks for creating this lib 🙏 

Can you give me an example of a shape of the `jwtClaims` return? I need to mock a return but, since I don't have a physical iPhone device here, I cant test my code to get a valid `appleToken` to send to your lib. Is it something like this?

![Screen Shot 2020-11-05 at 15 17 35](https://user-images.githubusercontent.com/12154623/98280355-0bd65880-1f7a-11eb-963f-d35a2a750681.png)

Thanks again, stefan!

**Comment by stefan-prokop-cz** [comment]
Hello, thank you so much!

Of course, here is an example of one of my old tokens (I also dont have physical iPhone or any other client to use :disappointed:):

```
{
  "kid": "ABC123",
  "alg": "RS256"
}
{
  "iss": "https://appleid.apple.com",
  "aud": "YOUR_PROJECT",
  "exp": 1586259128,
  "iat": 1586258528,
  "sub": "com.test",
  "c_hash": "2-BFgvjnxPyhvsl3B4LmMA",
  "email": "myemail@privaterelay.appleid.com",
  "email_verified": "true",
  "is_private_email": "true",
  "auth_time": 1586258528,
  "nonce_supported": true
}
```
I would like to add some mocks for testing and tests as well but I dont have enough time :disappointed: If you want you can open a PR for example with your mock or mock draft, it would be very helpful.
Thanks for your question, hope it helps :slightly_smiling_face:

**Comment by lcnogueira** [comment]
Hi, Stephan!

Thank you so much for answering so quickly! 😃 

Yeah, it helps a lot! 

I'm also kind of busy...actually, this is for something I'm testing for my job...but I'll try to get some time in order to open a PR here to help to improve this lib 👍 🙏 

Thanks again, Stephan!

**Comment by stefan-prokop-cz** [comment]
Hello @lcnogueira :wave: 

I found some time and added some tests and verification steps with refactoring. I also released new major version [`v2.0.0`](https://www.npmjs.com/package/verify-apple-id-token). Can you please try this version if you have some time? :pray: There are some breaking changes but not so big, you can find it in updated readme. Then let me know if everything works or report bugs.

I am asking you because I know you are implementing apple login and I have no opportunity to make a real test now :disappointed: 

Thank you so much! :slightly_smiling_face:

**Comment by stefan-prokop-cz** [comment]
it looks fine
